Dockerfile

```diff
@@ -0,0 +1,56 @@
+# 1: Build the exe
```

Comment:
I think docker files are pretty readable without comments, so I'd vote to delete them.